zebra-rpc/src/methods.rs

```diff
@@ -1108,6 +1140,88 @@ where
         .boxed()
     }
 
+    fn z_get_subtrees_by_index(
+        &self,
+        pool: String,
+        start_index: NoteCommitmentSubtreeIndex,
+        limit: Option<NoteCommitmentSubtreeIndex>,
+    ) -> BoxFuture<Result<GetSubtrees>> {
```

Comment:
This PR is ready for review, but we can't merge this new RPC until after the 1.2.0 release, because it doesn't have any data yet.

Reply:
I split the RPC method into another PR so this PR could merge.